src/lang/en-US.json

```diff
@@ -338,8 +338,8 @@
   "lane_role_3": "Off",
   "lane_role_4": "Jungle",
 
-  "lane_1": "Bottom",
-  "lane_2": "Middle",
+  "lane_1": "Bot",
```

Comment:
OH. It looks like those were added after my branch was created, so I don't have those changes yet.